Update what-is-license-compliance.md

rashi12122001 opened the pull request.

rashi12122001 committed:
Update what-is-license-compliance.md

Signed-off-by: rashi arora <74211043+rashi12122001@users.noreply.github.com>

avinal commented:
Thank you @rashi12122001 for the PR. I feel this topic is vast and this PR is an extremely summerized version of that. I expect to see at least 500 words. Please update the PR description as well.

rashi12122001 commented:
ok

rashi12122001 committed:
Update what-is-license-compliance.md

Hello team,
I would like to contribute in this project,i explored about license compliance and summerized it in 700 words
If it gets approved,it will be my first contribution to any organisation.

Cc: @avinal @GMishx @shaheemazmalmmd

Signed-off-by: rashi arora <74211043+rashi12122001@users.noreply.github.com>
